components/Item/ItemGrid.vue

```diff
@@ -107,11 +640,13 @@ export default Vue.extend({
   transform: translate(-50%, -50%);
 }
 
-@import '~vuetify/src/styles/styles.sass';
 .card-grid-container {
+  content-visibility: auto;
```

Comment:
This isn't supported in Safari, Firefox, or any mobile browsers excluding Chrome for Android

Reply:
I know, but it should be omitted in those browsers. My implementation will work with and without it, however, in those browsers where this property is available (Chrome as you mention) it will give performance benefits while scrolling really fast, as it will attempt to render first the contour of the card (instead of all the inners of it, which will only be rendered once they're 100% in viewport). This is really useful when scrolling really fast, as the user will see the cards contour as soon as possible but not further work will be done for the inners if the fast scroll continues.

Firefox and other browsers will render the inners of the cards as soon as they're swapped all the times, which is more taxing while scrolling fast.